src/EnergyPlus/RuntimeLanguageProcessor.cc

```diff
@@ -1360,314 +1368,65 @@ namespace RuntimeLanguageProcessor {
                     Token(NumTokens).Type = TokenOperator;
                 }
 
-                // First check for two character operators:  == <> <= >=
+                // parse an operator if found,
```

Comment:
So now you're just doing some extra stuff :upside_down_face: :+1:

Reply:
Yeah, sorry I couldn't bring myself to change the same code in 50 places when one simple function meant I only had to change it in one place.